overcast/django/apps/buildsvc/pkgbuild/__init__.py

```diff
@@ -13,12 +14,14 @@
 from ..models import BuildRecord
 
 class PackageBuilder(object):
-    def __init__(self, basedir, package_source, build_record):
+    def __init__(self, basedir, package_source, build_record,
+                 docker_build=True):
```

Comment:
As I asked in my original e-mail: What's the use case for non-dockerized builds?

I'm not going to use it. You're not going to use it. So we'll have a bunch of code noone uses. That's a guaranteed recipe for bitrot.